### DISCUSSION TITLE
Do not cache a specific html part

### DISCUSSION THREAD

**Discussion by jeanbelhache** [spatie/laravel-responsecache#445] [Q&A]
Hi,
is there a way to not caching a specific blade part ? 
For examples :
- a disclaimer on all pages whose acceptance is stored in session
- a section of the menu changes regularly according to the news written

Thank you all for your help

**Comment by lianmafutra** [comment]
you can use custom replacer in config/responsecache.php

'replacers' => [
\Spatie\ResponseCache\Replacers\CsrfTokenReplacer::class,
ReplacerCache::class,
],

create custom replacer class, and replace/remove string your html syntax,
